KebabBackend/database/migrations/0001_01_01_000000_create_users_table.php

```diff
@@ -13,10 +13,12 @@ public function up(): void
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('name')->unique();
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->boolean('is_Admin')->default(false);
+            $table->boolean('first_Login')->default(true);
```

Comment:
Why are the words admin and login capitalized here? Seems inconsistent with other variable names, I'd recommend sticking to one way of naming variables. Let's change these to is_admin and first_login or even is_first_login to make the boolean more explicit
```suggestion
            $table->boolean('is_admin')->default(false);
            $table->boolean('is_first_login')->default(true);
```

Reply:
Done https://github.com/MatWisie/KebabFinder-API/pull/24/commits/f9371bd79e6ceeed13412929c47b6854980c70e4